src/components/DraggableBox.vue

```diff
@@ -167,6 +175,37 @@ function onDrop(evt: DragEvent, list: number, snapItem?: Item) {
     }
   }
 }
+
+function handleSubmit() {
+  showResult.value = true;
+  if (listOne.value.length > 0) {
+    result.value = false;
+    return;
+  }
+
+  const allItemsMatch = listTwo.value.every((item) => {
+    const matchingSnapItem = snapItems.value.find(
+      (snapItem) => snapItem.id === item.id
+    );
+    if (!matchingSnapItem) {
+      result.value = false;
+      return;
+    }
+
+    return (
+      item.position.x === matchingSnapItem.position.x &&
+      item.position.y === matchingSnapItem.position.y
+    );
+  });
+
+  if (allItemsMatch) {
```

Comment:
Assuming this `allItemsMatch` returns a boolean, Can simplify it to

```ts
result.value = allItemsMatch
```

In fact, you could assign `result.value =listTwo.value.every(...)` directly instead right?

Reply:
resolved ! you're right it was awfully structured because I had a bunch of console.logs